app/dashboard/settings/auth/settings-auth-controller.ts

```
@@ -0,0 +1,35 @@
+  }
+  
+  /**
+   * Check if the page is still loading
+   * @param isAuthLoaded Whether auth data is loaded
+   * @param isUserLoading Whether user data is loading 
+   * @returns Boolean indicating if the page is still loading
+   */
+  static isLoading(isAuthLoaded: boolean, isUserLoading: boolean): boolean {
+    return !isAuthLoaded || isUserLoading;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)